en/app.ftl

```diff
@@ -487,13 +487,17 @@ settings-error-save-heading = Settings change failed
 settings-error-save-description = Your changes to settings were not saved due to a connection error. Please try again.
 # This is a warning displayed at the top of the settings page when server storage of mask labels and associated websites are turned off.
 settings-warning-collection-off-heading-2 = Email mask label function is disabled
+settings-warning-collection-off-heading-3 = Your email mask account names are turned off
 # This is a warning displayed at the top of the settings page when server storage of mask labels and associated websites are turned off.
 settings-warning-collection-off-description-2 = { -brand-name-relay } is not currently allowed to collect the data showing the sites where you’ve generated and used email masks. You can change this in “Settings” under “Data Collection.”
+settings-warning-collection-off-description-3 = { -brand-name-relay } is not currently allowed to label your email masks with the websites where you’ve used them. To change this, enable account names for email maks under “Privacy.”
```

Comment:
Should this end with `"Privacy".` instead of `"Privacy."`?

Reply:
Probably a Tony/Mel question - let me ask them. This is what we got from them: 
<img width="1155" alt="image" src="https://user-images.githubusercontent.com/3924990/210111966-e7a0bf60-360f-4b08-b928-f1a0f38551b7.png">

Reply:
Talked to Mel and Tony - both agree to leave it as is, no link needed as privacy link is below on the page (footer)

Reply:
You might have discussed offline, but I think Kaitlyn meant that the period isn't part of the label and thus shouldn't be inside the quotes? Also, I think it should be "masks" instead of "maks" :)

```suggestion
settings-warning-collection-off-description-3 = { -brand-name-relay } is not currently allowed to label your email masks with the websites where you’ve used them. To change this, enable account names for email masks under “Privacy”.
```

Reply:
@Vinnl Mel wants it inside the quotes. I'll update the masks bit - good catch!

Reply:
I think I replied to this with the wrong info as I believe Kaitlyn had also asked about a link to the privacy page, lol.